environments/ithc/ithc-platform-hmcts-net.yml

```diff
@@ -94,6 +94,9 @@ cname:
   - name: idam-testing-support-api
     record: idam-testing-support-api-ithc.service.core-compute-ithc.internal.
     ttl: 300
+  - name: idam-hmcts-access
```

Comment:
Should it be `idam-hmcts-access` or just `hmcts-access`?

Reply:
I guess the prefix lets other people know this is an idam managed project just by its name?

Reply:
That's the CNAME record though, isn't it?

I believe our ingress host name is `hmcts-access.ithc.platform.hmcts.net`.